code/modules/surgery/organic_steps.dm

```diff
@@ -138,3 +138,37 @@
 		"<span class='notice'>[user] drills into [target]'s [parse_zone(target_zone)]!</span>",
 		"<span class='notice'>[user] drills into [target]'s [parse_zone(target_zone)]!</span>")
 	return ..()
+
+//pick bone
```

Comment:
```suggestion

```
Not very informative.

Reply:
all of the surgery steps are commented on like this so they can be seen better when you're designing a new surgery and need to find which steps it should have.